buildSrc/build.gradle

```diff
@@ -0,0 +1,12 @@
+apply plugin: 'groovy'
+apply plugin: 'java'
+
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
```

Comment:
does this change is required by all plugin? if so, could you add related issue/pr.

Reply:
We are using a third party plugin to spawn and kill process in doctest module. The plugin is around 5 years old and there is no active development. As part of Gradle 7 upgrade spawn plugin is throwing errors. As part of fix, imported the plugin code to buildSrc and upgraded to Gradle 7.[The only way for upgrading unmaintained third party plugins]. I couldn't find another well maintained plugin to replace.

Reply:
https://github.com/opensearch-project/sql/issues/303.  Will add to the description.

Reply:
thanks. could we create an issue to tracking move this dependency later. ideally, we should not depend on it.